src/Features/Core/Portable/InlineHints/AbstractInlineParameterNameHintsService.cs

```diff
@@ -72,17 +74,23 @@ public async Task<ImmutableArray<InlineHint>> GetInlineHintsAsync(Document docum
 
             void AddHintsIfAppropriate(SyntaxNode node)
             {
+                var syntaxFacts = document.GetRequiredLanguageService<ISyntaxFactsService>();
```

Comment:
Will remove, was going to use it to get the argument name but misinterpreted the GetNameOfArgument method